### ISSUE TITLE
Kamailio 5.7 TLS fail to compile

### ISSUE THREAD

**Issue by iliesh** [kamailio/kamailio#3796]
### Description

Unable to upgrade Kamailio 5.7 on CentOS7

Error:
```
tls_mod.c: In function ‘fork_child’:
tls_mod.c:348:2: error: ‘for’ loop initial declarations are only allowed in C99 mode
  for(int k = 0; k < 16; k++) {
  ^
tls_mod.c:348:2: note: use option -std=c99 or -std=gnu99 to compile your code
make[2]: *** [tls_mod.o] Error 1
make[1]: *** [modules] Error 1
```

### Additional Information

  * **Kamailio Version** - output of `kamailio -v`
Current installed version:
```
version: kamailio 5.7.4 (x86_64/linux) 2e6e87
```

**Comment by henningw** [comment]
Thanks for the report. There are also some issues like this in file_out. @xkaraman could you please have a look?

**Comment by xkaraman** [comment]
Fixed in #3797
